[2025-02-10T15:33:06.540+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T15:33:06.667+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.verify_file_reliability scheduled__2025-02-10T15:32:00+00:00 [queued]>
[2025-02-10T15:33:06.675+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.verify_file_reliability scheduled__2025-02-10T15:32:00+00:00 [queued]>
[2025-02-10T15:33:06.676+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T15:33:06.688+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): verify_file_reliability> on 2025-02-10 15:32:00+00:00
[2025-02-10T15:33:06.694+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6946) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T15:33:06.696+0000] {standard_task_runner.py:72} INFO - Started process 6948 to run task
[2025-02-10T15:33:06.696+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag', 'verify_file_reliability', 'scheduled__2025-02-10T15:32:00+00:00', '--job-id', '367', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmp93ifkag9']
[2025-02-10T15:33:06.697+0000] {standard_task_runner.py:105} INFO - Job 367: Subtask verify_file_reliability
[2025-02-10T15:33:06.737+0000] {task_command.py:467} INFO - Running <TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.verify_file_reliability scheduled__2025-02-10T15:32:00+00:00 [running]> on host b6443be51f35
[2025-02-10T15:33:06.815+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag' AIRFLOW_CTX_TASK_ID='verify_file_reliability' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T15:32:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T15:32:00+00:00'
[2025-02-10T15:33:06.817+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T15:33:06.817+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T15:33:06.818+0000] {logging_mixin.py:190} INFO - Current task name:verify_file_reliability state:running start_date:2025-02-10 15:33:06.668002+00:00
[2025-02-10T15:33:06.818+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag and current dag run status:running
[2025-02-10T15:33:06.819+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T15:33:06.819+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-10T15:33:06.820+0000] {logging_mixin.py:190} INFO - camarche22 /opt/***/files/in ['.csv'] ['utf-8', 'ISO-8859-1', 'Windows-1252']
[2025-02-10T15:33:06.829+0000] {my_directory_and_file_checker.py:241} ERROR - ‚ùå Erreur d'encodage (utf-8) pour √âcritures comptables article (10).csv: 'utf-8' codec can't decode byte 0xb0 in position 1: invalid start byte
[2025-02-10T15:33:06.850+0000] {logging_mixin.py:190} INFO - 
üìÇ Contenu du fichier (ISO-8859-1): √âcritures comptables article (10).csv
[2025-02-10T15:33:06.859+0000] {logging_mixin.py:190} INFO -   Date comptabilisation      N¬∞ lot Type √©criture N¬∞ document  N¬∞ s√©quence
0            16/12/2024         NaN         Vente   BL1900145        15735
1            16/12/2024         NaN         Vente      107012        15734
2            11/01/2025  16LABCDE24    Production   RPO000460        15733
3            11/01/2025  16LABCDE24    Production   RPO000460        15732
4            11/01/2025  16LABCDE24    Production   RPO000460        15731
[2025-02-10T15:33:08.003+0000] {logging_mixin.py:190} INFO - ‚úÖLignes  ('‚úÖ Le fichier respecte les colonnes attendues et les types de donn√©es.', True) 10923
[2025-02-10T15:33:08.007+0000] {logging_mixin.py:190} INFO - merci
[2025-02-10T15:33:08.016+0000] {base.py:84} INFO - Retrieving connection 'mysql_conn'
[2025-02-10T15:33:08.029+0000] {logging_mixin.py:190} INFO - (('attendancedb',), ('caprostockdb',), ('information_schema',), ('mysql',), ('netvision_db',), ('performance_schema',), ('pointing_db',), ('posgpt_db',), ('pythondb',), ('sys',), ('testimportdb',))
[2025-02-10T15:33:08.070+0000] {logging_mixin.py:190} INFO - ‚ö†Ô∏è Donn√©es supprim√©es avant insertion.
[2025-02-10T15:33:08.569+0000] {sql.py:691} INFO - Loaded 1000 rows into temp_ecc so far
[2025-02-10T15:33:09.178+0000] {sql.py:691} INFO - Loaded 2000 rows into temp_ecc so far
[2025-02-10T15:33:09.781+0000] {sql.py:691} INFO - Loaded 3000 rows into temp_ecc so far
[2025-02-10T15:33:10.417+0000] {sql.py:691} INFO - Loaded 4000 rows into temp_ecc so far
[2025-02-10T15:33:11.003+0000] {sql.py:691} INFO - Loaded 5000 rows into temp_ecc so far
[2025-02-10T15:33:11.643+0000] {sql.py:691} INFO - Loaded 6000 rows into temp_ecc so far
[2025-02-10T15:33:12.308+0000] {sql.py:691} INFO - Loaded 7000 rows into temp_ecc so far
[2025-02-10T15:33:12.949+0000] {sql.py:691} INFO - Loaded 8000 rows into temp_ecc so far
[2025-02-10T15:33:13.612+0000] {sql.py:691} INFO - Loaded 9000 rows into temp_ecc so far
[2025-02-10T15:33:14.262+0000] {sql.py:691} INFO - Loaded 10000 rows into temp_ecc so far
[2025-02-10T15:33:14.905+0000] {sql.py:694} INFO - Done loading. Loaded a total of 10923 rows into temp_ecc
[2025-02-10T15:33:14.907+0000] {logging_mixin.py:190} INFO - connexion sql reussie True
[2025-02-10T15:33:14.937+0000] {logging_mixin.py:190} INFO - ‚ùå Erreur lors du d√©placement du fichier : [Errno 13] Permission denied: '/opt/***/files/in/√âcritures comptables article (10).csv'
[2025-02-10T15:33:14.941+0000] {python.py:240} INFO - Done. Returned value was: import_temp_ecc_to_ecc
[2025-02-10T15:33:14.941+0000] {branch.py:38} INFO - Branch into import_temp_ecc_to_ecc
[2025-02-10T15:33:14.942+0000] {skipmixin.py:230} INFO - Following branch ('import_temp_ecc_to_ecc',)
[2025-02-10T15:33:14.952+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-10T15:33:14.978+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T15:33:14.979+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag, task_id=verify_file_reliability, run_id=scheduled__2025-02-10T15:32:00+00:00, execution_date=20250210T153200, start_date=20250210T153306, end_date=20250210T153314
[2025-02-10T15:33:14.989+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T15:33:14.990+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T15:33:14.990+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag queued_at:2025-02-10 15:33:00.761307+00:00
[2025-02-10T15:33:14.991+0000] {logging_mixin.py:190} INFO - Task hostname:b6443be51f35 operator:BranchPythonOperator
[2025-02-10T15:33:15.017+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T15:33:15.049+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-10T15:33:15.053+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
